lib/board.rb

```diff
@@ -0,0 +1,39 @@
+# rubocop: disable Metrics/AbcSize
+
+# rubocop: disable Metrics/CyclomaticComplexity
+
+# rubocop: disable Metrics/PerceivedComplexity
+
+class Board
+  attr_accessor :board
+  def initialize(board)
+    @board = board
+  end
+
+  def display_board
+    puts " #{@board[0]} | #{@board[1]} | #{@board[2]} "
+    puts ' ----------- '
+    puts " #{@board[3]} | #{@board[4]} | #{@board[5]} "
+    puts ' ----------- '
+    puts " #{@board[6]} | #{@board[7]} | #{@board[8]} "
+  end
```

Comment:
- [ ] You shouldn't use `puts/print` statements outside of the `bin/main.rb` file. 
screenshot of the requirements 👇 

![Screen Shot 2020-08-28 at 1 08 02 AM](https://user-images.githubusercontent.com/48804258/91503505-ed454880-e8ca-11ea-97d3-87e1a34c0c3c.png)

Reply:
:100: